📚 DOCS: Added language about read-only codeMirror default (#287)
executablebooks/thebe: docs/configure.rst

Mark a code cell as read-only
=============================

If you would like a code cell to be runnable by Thebe, but not *editable* by the user, you
may mark it as "read-only" with the following syntax:

.. code-block:: html

   <pre data-executable data-readonly>print("I cannot be modified")</pre>

Users will not be able to modify the code once Thebe is activated, though they can still
press the "run" button to see the outputs.

> Alternatively, you can use the codeMirrorConfig in order to set the default as read-only for all cells:

Reviewer: ```suggestion
**To set all cells as read-only by default**, use the following `thebe` configuration:
```

.. code:: javascript

   codeMirrorConfig: {
       readOnly: true
   }

> And when read-only as the default, you can still override this setting for individual cells using `data-readonly`:

Reviewer: ```suggestion
This uses codeMirror to mark all cells as read-only. If you are using this setting and would like to
manually mark cells as editable, you can override the codeMirror configuration for individual cells using `data-readonly="false"`. For example:
```

Mark a code cell as read-only
=============================

If you would like a code cell to be runnable by Thebe, but not *editable* by the user, you
may mark it as "read-only" with the following syntax:

.. code-block:: html

   <pre data-executable data-readonly>print("I cannot be modified")</pre>

Users will not be able to modify the code once Thebe is activated, though they can still
press the "run" button to see the outputs.

**To set all cells as read-only by default**, use the following `thebe` configuration:

.. code:: javascript

   codeMirrorConfig: {
       readOnly: true
   }

This uses codeMirror to mark all cells as read-only. If you are using this setting and would like to

> manually mark individual cells as editable, you can override the codeMirror configuration for a cell using `data-readonly="false"`. For example:

Author: I can't figure out why \`data-readonly="false"\` isn't rendering as `data-readonly="false"` here for inline code. Any ideas? I think this is the last outstanding issue for this PR.

Reviewer: ah my bad - it was because I used a single backtick and rST requires two backticks by default

> manually mark individual cells as editable, you can override the codeMirror configuration for a cell using `data-readonly="false"`. For example:

Reviewer: ```suggestion
manually mark individual cells as editable, you can override the codeMirror configuration for a cell using ``data-readonly="false"``. For example:
```
